src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1217,6 +1231,23 @@ public override SyntaxNode InsertNamespaceImports(SyntaxNode declaration, int in
             }
         }
 
+        private static SyntaxList<UsingDirectiveSyntax> CopyBanner(CompilationUnitSyntax cu, SyntaxList<UsingDirectiveSyntax> usingsToInsert)
+        {
+            // First. Strip any pp directives or banners on the compilation unit.  They
+            // have to stay at the top of the list.
+            IEnumerable<SyntaxTrivia> banner = cu.GetLeadingBannerAndPreprocessorDirectives();
+
+            // Now, we want to remove any blank lines from the new first node and then reattach the banner. 
+            var finalFirstNode = usingsToInsert[0];
+            finalFirstNode = finalFirstNode.GetNodeWithoutLeadingBlankLines();
```

Comment:
I copied this from http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/Organizing/Organizers/MemberDeclarationsOrganizer.cs,97.

Whitespace is removed from the old first node. Later, that leading whitespace is then added to the new first node (method InsertNamespaceImports at line 1221)